src/ParseFile.js

```diff
@@ -123,6 +123,7 @@ class ParseFile {
           type: specifiedType,
         };
       } else if (typeof Blob !== 'undefined' && data instanceof Blob) {
+        this._data = data;
```

Comment:
I believe this should be removed since _data is always base64. You are setting a `Blob` here

Reply:
Thanks. Let me correct this.

Reply:
This has been removed.

Reply:
Please mark this conversation as resolved if that's what is required.